action.yml

```diff
@@ -32,6 +32,26 @@ inputs:
       for the surest of choices.
     required: false
     default: Changelog
+  version-tab:
+    description: |
+      Configuration how should the dropdownlist appear for multiple versions.
+      It should be set as an ASCII text representation of an R list object 
+      Example:
+      """
+      list(config = list(
+              tooltip = list(
+                main = "Tooltip for main branch"
+              ), 
+              text = list( 
+                main = "main branch"
+              )
+            ))
+      """
+      String should be quoted with " sign
```

Comment:
Either add this in the README or remove it from here, otherwise the docs are inconsistent

Reply:
Added into README.md